app/durableObjects/ChatRoom.server.ts

```diff
@@ -170,6 +172,10 @@ export class ChatRoom extends Server<Env> {
 		const roomStateMessage = JSON.stringify(roomState)
 
 		for (const connection of this.getConnections()) {
+			// skip connection if it's not OPEN
+			if (connection.readyState !== WebSocket.OPEN) {
```

Comment:
It seems like partyserver's `InMemoryConnectionManager` is behaving differently from the `HibernatingConnectionManager`, and connections aren't being removed after a failed attempt to broadcast, so we need to manually skip them for now.